analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -144,9 +146,11 @@ private static void DumpActualDiagnostics(string languageVersion, IEnumerable<Di
             var compilationOptions = compilation.Options.WithSpecificDiagnosticOptions(supportedDiagnostics);
             var actualToken = cancellationToken ?? CancellationToken.None;
 
+            var analyzerOptions = string.IsNullOrWhiteSpace(sonarProjectConfigPath)? null : TestHelper.CreateOptions(sonarProjectConfigPath);
```

Comment:
_Double nitpick:_ EOL before is not needed, previous block build opions. There's a space missing before `?` operator
```suggestion
            var analyzerOptions = string.IsNullOrWhiteSpace(sonarProjectConfigPath) ? null : TestHelper.CreateOptions(sonarProjectConfigPath);
```